backend/swagger/v1/swagger.yaml

```diff
@@ -349,7 +356,7 @@ paths:
               $ref: '#/components/schemas/CreateAnswersSurveyRequestBody'
   '/api/v1/answers_surveys/{id}':
     get:
-      summary: Get Answers Survey
+      summary: List a single Answers Survey by ID
```

Comment:
List word is associated to arrays, so I would go with `Get Answers Survey by ID` or `Retrieve Answers Survey by ID`